### DISCUSSION TITLE
<p-Captcha>  tag is unknow element

### DISCUSSION THREAD

**Discussion by AliNikfar** [primefaces/community#721] [PrimeNG] [answered]
hi , i'm trying to add captcha to one of my components using primeng  captcha  .
but when i'm trying to add import  "{CaptchaModule} from 'primeng/captcha' " to my appmodule.ts file , angular says cannot find the path 
and can't know <p-captcha> as known element !
a lot of  refrencess refer bootstrap but i used primeng in my whole project what i have to do ?
what i have to do !????

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Captcha component has been deprecated since PrimeNG v15 and it's not available in v15+ versions. Unfortunately, you cannot use the component with PrimeNG 15,16 and 17. We suggest using the captcha component provided by Google captcha or any other captcha services you use.

**Reply by AliNikfar** [reply]
it means have to change all my project UI framework to something else like bootstrap or another way is 
use backend captcha  and just show "captchaed image" in the component?!
or can u suggest another way  ??
i used primeng  on whole of project components and UI framework changing  takes a lot times   :(
